### PR TITLE
Fix ignore openapi doc page bug

### PR DESCRIPTION
**Pull request by chrisho**
Fix bugs:
- Fix shell syntax issue.
- Fix ignore openapi doc page bug.

### CODE REVIEW

**Review by futuretea — APPROVED** [review]
LGTM, thanks.

File: scripts/ignore-openapi [hunk 1]
```diff
@@ -39,24 +39,26 @@ if echo "$FILES" | grep -q "$API_DIRECTORY/*.mdx"; then
     exit 0
 fi
 
-# ignore if the docs/api folder not exists.
-if [ -d "docs/$API_DIRECTORY" ]; then
-    # if no any change then don't build the openapi documents.
-    rm -rf docs/$API_DIRECTORY/*.mdx
-    cat << EOF > docs/$API_DIRECTORY/sidebar.js
-module.exports = [];
+yarn clean-api-docs
```
Comment (guangbochen) [inline scripts/ignore-openapi:42]: can u please add a comment to explain what the following script is doing, thanks.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
lgtm! thanks!

**Pull request merged**